src/xia2/Driver/DriverHelper.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import annotations
 
+import logging
 import os
 import random
 import signal
 import stat
 import string
 
+logger = logging.getLogger("xia2.Driver.DriverHelper")
+
```

Comment:
This is leftover from old implementation?
```suggestion
import os
import random
import signal
import stat
import string

```

Reply:
Yes, it was residual from `windows_resolve`